lualib/lua_scanners/icap.lua

```diff
@@ -85,6 +88,10 @@ local function icap_config(opts)
     icap_conf.servers,
     icap_conf.default_port)
 
+  if icap_conf.user_agent == "extended" then
+    icap_conf.user_agent = string.format("Rspamd/%s-%s (%s)", rspamd_version('main'), rspamd_version('id'), rspamd_util.get_hostname())
```

Comment:
Maybe also add the current uid. That would help to debug issues on the icap-server side.

`string.sub(task:get_uid(), 1,6)`

Reply:
Excellent idea! I have added that one just now. User Agent as tested would be: 

`User-Agent: Rspamd/2.6-release (localhost.localdomain/ab5ab0)`